Build fails if "bin" folder does not exist before compilation.

After upgrading to release 1.2.1 of TSLint the build fails on our build server.
This seems to be because the TSLint build step is run before compilation in our build, so the "bin" folder is not yet created.

```
"C:\projects\foo\MsBuildSettings.msbuild" (default target) (1) ->
"C:\projects\foo\foo.sln" (Clean;Rebuild target) (2) -> "C:\projects\ct\foo\Code\Web\foo\foo.Web.csproj" (Rebuild target) (20:2) -> 
(TSLint target) -> 
C:\projects\foo\packages\TSLint.MSBuild.1.2.1\build\TSLint.MSBuild.targets(60,5): 
error MSB3371: The file "bin\\foo.Web.TSLint.timestamp.txt" cannot be created. 
Could not find a part of the path 'C:\projects\foo\Code\Web\foo.Web\bin\foo.Web.TSLint.timestamp.txt'. 
[C:\projects\foo\Code\Web\foo.Web\foo.Web.csproj]
```

--- a/src/build/TSLint.MSBuild.targets
+++ b/src/build/TSLint.MSBuild.targets
@@ -57,6 +57,8 @@
 
     <Error Condition="'$(TSLintFileListDisabled)' == 'true' And '$(TSLintProject)' == ''" Text="You disabled file listing on the command line using TSLintFileDisabled, but did not specify a project file with TSLintProject." />
 
+    <MakeDir Directories="$(OutputPath)" />
+    
     <Touch
       AlwaysCreate="true"
       Files="$(TSLintTimestampFile)" />

